cmake/Config.cmake

```diff
@@ -115,6 +100,30 @@ elseif(MSVC)
         set(SFML_MSVC_VERSION 12)
     elseif(MSVC_VERSION EQUAL 1900)
         set(SFML_MSVC_VERSION 14)
+    elseif(MSVC_VERSION LESS_EQUAL 1919)
+        set(SFML_MSVC_VERSION 15)
+    elseif(MSVC_VERSION LESS_EQUAL 1929)
+        set(SFML_MSVC_VERSION 16)
+    endif()
+elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER MATCHES "clang[+][+]")
```

Comment:
Is the check for `CMAKE_CXX_COMPILER_ID` not enough? (e.g. https://stackoverflow.com/a/10055571 claims it is)

Reply:
Could very well be. For a long time `CMAKE_CXX_COMPILER_ID` was not a thing and then after it was an undocumented thing, but now I believe it's fully documented, so the other part could be dropped.